fuse_constraints/include/fuse_constraints/normal_prior_orientation_3d_euler_cost_functor.hpp

```diff
@@ -83,12 +83,12 @@ class NormalPriorOrientation3DEulerCostFunctor
    *
    * @param[in] A The residual weighting matrix, most likely the square root information matrix. Its
    *              order must match the values in \p axes.
-   * @param[in] b The orientation measurement or prior. Its order must match the values in \p axes.
+   * @param[in] b The orientation measurement or prior.
```

Comment:
Why this comment is removed? No more valid?

Reply:
My changes in this file came from tests that I tried some time ago and are not valid. I will revert it to the previous commit